### DISCUSSION TITLE
[Icon Request] Classic Side Panel icon

### DISCUSSION THREAD

**Discussion by tomo-local** [tailwindlabs/heroicons#1207] [Ideas]
A classic side pannel icon is very useful for forms. 😁
ex) [side-panel](https://thenounproject.com/browse/icons/term/side-panel/)

**Comment by TheoMcCabe** [comment]
Agree. 

 Very popular with AI apps now, as they often have closable sidebar chat panels. 
 
You can see an example from chat gpt here: 
 
<img width="230" alt="Screenshot 2025-01-04 at 11 00 13" src="https://github.com/user-attachments/assets/e125abbb-be77-4a58-a886-f819c2e48b72" />
<img width="430" alt="Screenshot 2025-01-04 at 11 00 08" src="https://github.com/user-attachments/assets/98b27baa-1ddd-4449-b9e6-4dc180b04797" />
<img width="51" alt="Screenshot 2025-01-04 at 11 00 48" src="https://github.com/user-attachments/assets/5ebd43df-ec5e-4d7c-8d6c-226fc9f5f0f8" />

**Comment by ndri** [comment]
I added `sidebar-left` and `sidebar-right` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons.

* https://sidekickicons.com/?icon=sidebar-left
* https://sidekickicons.com/?icon=sidebar-right

**Comment by tomo-local** [comment]
@ndri 
Great additions! These icons will be really useful alongside Heroicons. Thanks for sharing! 
By the way, is there a chance they will be added to Heroicons in the future, or will they likely remain exclusive to Sidekickicons? 🤔

**Reply by ndri** [reply]
That's for the Tailwind/Heroicons team to decide, not me. 😄
